collect/collect.go

```diff
@@ -498,6 +525,40 @@ func (i *InMemCollector) processSpan(sp *types.Span) {
 		trace.SendBy = time.Now().Add(timeout)
 		trace.RootSpan = sp
 	}
+	i.Metrics.Increment("span_processed")
```

Comment:
Duplicate incrementing again.

Reply:
Hmm. I hope I didn't screw up a merge.

Reply:
Ah, I reverted it when I split this into two PRs. No big deal. Thanks for catching it again!